Config.toml

```diff
@@ -14,3 +14,7 @@ waiting_period = 30
 retry_request_config = false
 # User creation retry attempts (not related to retry_request_config)
 user_creation_retry_attempts = 5
+# Max throughput allowed for users registration 
+user_creation_throughput = 10
+# Max throughput allowed for room creation 
+room_creation_throughput = 20
```

Comment:
This is not being used currently, I guess it must be a WIP or paving the way for a new PR.

Reply:
It's being used now, it was a WIP